callcommands/sv_callcommands.lua

```diff
@@ -162,6 +171,13 @@
                         postal = PostalsCache[source]
                     end
                     if Config.apiSendEnabled then
+						local callerApiId = nil
+						-- Set the caller api to something so we don't get an unhandled exception
+						if source ~= nil then
+							callerApiId = GetIdentifiers(source)[Config.primaryIdentifier] 
+						else 
+							callerApiId = 0
```

Comment:
This will break checks in dispatchnotify that look for a not-nil callerApiId. It should be left nil if not needed.

Reply:
The problem is generating calls from E.G smart fires do not have a source. So, you would need this to prevent an unhandled exception.